src/js/controllers/NumbersCtrl.js

```diff
@@ -325,6 +322,8 @@ function NumbersCtrl($scope) {
         unique_Org = org_names.filter(onlyUnique);
         $scope.sortedCountries = unique;
         $scope.sortedOrg = unique_Org;
+        var ONE_WEEK = 1000 * 60 * 60 * 24 * 7;
```

Comment:
Actually this should have been deleted. It's not being used anywhere. Ill fix this